### PR TITLE
Update HtmlEditor demos

### PR DESCRIPTION
**Pull request by DokaRus**
- Update "Overview" demo
image

- Add "Tables" demo
image

### CODE REVIEW

File: JSDemos/Demos/HtmlEditor/Overview/jQuery/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $(function() {
     var editor = $(".html-editor").dxHtmlEditor({
-        height: 550,
+        height: 600,
```
Comment (alexander-kotov-dx) [inline JSDemos/Demos/HtmlEditor/Overview/jQuery/index.js]: Those changes are for Jquery only?

Comment (DokaRus) [reply JSDemos/Demos/HtmlEditor/Overview/jQuery/index.js]: Nope, I'll update all approaches

**Comment by dxbykov** [comment]
1. If we add tables to the Overview demo, what's the reason to have a separate Tabels demos?
2. Table toobar items take too much space. I would expect them in a drop-down or in a context menu.
3. Let's use better demo data. 'task 1, task 2' doesn't look good.

**Comment by DokaRus** [comment]
Overview demo has been updated: 
image

+ We're marked HtmlEditor - Tables demo as `Hidden`.

>If we add tables to the Overview demo, what's the reason to have a separate Tabels demos?

1. We can stay with added table operations in this demo (but without a table markup) and not add a new demo
2. We can add a simple table markup to the `Overview` demo and not add a new demo
3. We can remove table operations from toolbar and demonstrate `Tables` feature in the standalone demo.
4. 1 or 2 + add new `Tables` demo.

I've discussed it with @San4es and @DX-man, and we are stuck between №1 and №3.

>I would expect them in a drop-down or in a context menu.

Agreed. I've been thinking about showing context-related operations in overlay on focusing\selection related content - images, tables and other embed content [like the most part of other vendors do]

>Let's use better demo data. 'task 1, task 2' doesn't look good.

Ok

File: JSDemos/Demos/HtmlEditor/Tables/Angular/app/app.component.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        BrowserModule,
+        DxHtmlEditorModule
+    ],
+    declarations: [AppComponent],
+    bootstrap: [AppComponent]
+})
+
+export class AppModule { }
+
+platformBrowserDynamic().bootstrapModule(AppModule);
```
Comment (San4es) [inline JSDemos/Demos/HtmlEditor/Tables/Angular/app/app.component.ts]:
```suggestion
platformBrowserDynamic().bootstrapModule(AppModule);

```

**Review by San4es — DISMISSED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**